0x/Assets/ERC721Asset.cs

```diff
@@ -0,0 +1,76 @@
+﻿
+using Nethereum.ABI;
+using Nethereum.ABI.Encoders;
+using Nethereum.Util;
+using System;
+using System.Linq;
+using System.Numerics;
+using ZeroX.Utilities;
+
+namespace ZeroX.Assets
+{
+
+    /// <summary>
+    /// ERC721 Asset
+    /// </summary>
+    public class ERC721Asset : TokenAsset
+    {
+
+        private static readonly byte[] _erc721AssetHeader = new byte[] { 0x02, 0x57, 0x17, 0x92 };
```

Comment:
Also, I would put this in some contants file.

Reply:
I would not, currently this is only used in this class.